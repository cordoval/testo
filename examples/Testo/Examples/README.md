Some description of the awesome code:

<?php
echo 'hello world!';
?>