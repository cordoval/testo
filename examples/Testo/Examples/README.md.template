Some description of the awesome code:

<?php
@testo Testo\Examples\Hello world
?>