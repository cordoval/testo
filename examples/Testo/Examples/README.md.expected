Some description of the awesome code:

<?php
@testo Testo\Examples\Hello world {
echo 'hello world!';
@testo 1abb62086e2cc233ede1e19de3a8e5f6 }
?>